openedx/adg/lms/applications/models.py

```diff
@@ -191,7 +190,9 @@ def prereq_course_scores(self):
         Returns:
             list: Prereq course name and score pairs
         """
-        prereq_course_overviews = CourseOverview.objects.filter(id__in=CourseMeta.prereq_course_ids.all())
+        prereq_course_overviews = CourseOverview.objects.filter(
```

Comment:
Good idea to display all courses in each group for now! In future probably we will be showing the ones user has enrolled in or the ones user has passed